Core/Inc/DynamicControls/longitudinal_control.h

```diff
@@ -9,6 +9,7 @@
 
 #include <stdint.h>
 #include "PID.h"
+#include "CMSIS_EXTRA/global_variables_handler.h"
```

Comment:
```suggestion
#include "CMSIS_extra/global_variables_handler.h"
```
por isso tá falhando a compilação rs